.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -1,7 +1,7 @@
 name: Auto Approve Dependency PRs
 on:
   schedule:
-    - cron: '*/30 * * * *'
+    - cron: '*/5 * * * *'
```

Comment:
I asked Nate for this change (rather than putting up a 1 line PR). I want this workflow to run more often.